rpcserver.go

```diff
@@ -1827,18 +1856,57 @@ func KeypoolRefill(w *Wallet, chainSvr *chain.Client, icmd btcjson.Cmd) (interfa
 	return nil, nil
 }
 
-// GetNewAddress handlesa getnewaddress request by returning a new
-// address for an account.  If the account does not exist or the keypool
-// ran out with a locked wallet, an appropiate error is returned.
+// CreateNewAccount handles a createnewaccount request by creating and
+// returning a new account. If the last account has no transaction history
```

Comment:
The returned result is always nil (or JSON null).  If it's not already, can we make the default name for new accounts the string of the account number, and return that?